lib/cli/commands/listpeers.ts

```diff
@@ -1,11 +1,74 @@
 import { callback, loadXudClient } from '../command';
 import { Arguments } from 'yargs';
-import { ListPeersRequest } from '../../proto/xudrpc_pb';
+import colors from 'colors/safe';
+import Table, { HorizontalTable } from 'cli-table3';
+import { generateHeaders } from '../utils';
+import { ListPeersRequest, ListPeersResponse, Peer } from '../../proto/xudrpc_pb';
+
+const HEADERS = [
+  'address',
+  'public key',
```

Comment:
Could we call this `node key` actually? I'm thinking it will further differentiate it from the lnd pub keys.